src/Resources/skeleton/doctrine/Repository.tpl.php

```diff
@@ -47,7 +47,7 @@ public function upgradePassword(<?= sprintf('%s ', $password_upgrade_user_interf
 //        return $this->createQueryBuilder('<?= $entity_alias; ?>')
 //            ->andWhere('<?= $entity_alias; ?>.exampleField = :val')
 //            ->setParameter('val', $value)
-//            ->orderBy('<?= $entity_alias; ?>.id', 'ASC')
+//            ->orderBy('<?= $entity_alias; ?>.id', Criteria::ASC)
```

Comment:
in general terms, I'm not opposed to using a `const` or `enum` in place of strings. My only concern would be, are `doctrine/common` / `doctrine/common/collections` classes on the chopping block in `dbal` 4||5. E.g. where we would have to revert this behavior in the immediate future.

 I took a quick peak in `dbal/src/QueryBuilder` and did not see any evidence of such a possibility. But did not dig through either of the repos for open prs/issues on the matter.

If that's not an issue, green tests - I'm all for it.